client/scripts/views/pages/view_proposal/index.ts

```diff
@@ -93,10 +93,12 @@ const ProposalHeader: m.Component<{
 
     // Original posters have full editorial control, while added collaborators
     // merely have access to the body and title
-    const isAuthor = (app.user.activeAccount?.address === proposal.author
-          && app.user.activeAccount?.chain.id === (proposal as OffchainThread).authorChain);
+    const { activeAccount } = app.user;
+    const authorChain = (proposal instanceof OffchainThread) ? proposal.authorChain : app.activeId();
```

Comment:
we should just add `chain` to Proposal object in the future, but this seems fine for now